lib/utils.sh

```diff
@@ -42,8 +65,11 @@ list_installed_versions() {
   local plugin_path
   plugin_path=$(get_plugin_path "$plugin_name")
 
+  local install_dir
+  install_dir="$(asdf_data_dir)/installs"
```

Comment:
I don't think this variable is needed. This could be combined with line 72 and assigned to `plugin_installs_path`.

Reply:
I was trying to make it more semantic, I can merge it with L72 :+1: